akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -241,6 +241,8 @@ final class Flow[-In, +Out, +Mat](private[stream] override val module: Module)
 
   /** Converts this Scala DSL element to it's Java DSL counterpart. */
   def asJava: javadsl.Flow[In, Out, Mat] = new javadsl.Flow(this)
+
+  override def toString = s"""Flow(${module.toString})"""
```

Comment:
isn't

~~~scala
override def toString = s"""Flow(${module.toString})"""
~~~

equivalent to

~~~scala
override def toString = s"""Flow(${module})"""
~~~